common/constants.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed
+ * under the Apache License Version 2.0.
+ *
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2021 Datadog, Inc.
+ */
+
+export const LAYER_PREFIX = "DatadogLayer";
+export const EXTENSION_LAYER_PREFIX = "DatadogExtension";
+export const DD_ACCOUNT_ID = "464622532012";
+export const DD_GOV_ACCOUNT_ID = "002406178527";
+export const DD_HANDLER_ENV_VAR = "DD_LAMBDA_HANDLER";
+export const PYTHON_HANDLER = "datadog_lambda.handler.handler";
+export const JS_HANDLER_WITH_LAYERS = "/opt/nodejs/node_modules/datadog-lambda-js/handler.handler";
+export const JS_HANDLER = "node_modules/datadog-lambda-js/dist/handler.handler";
+export const SUBSCRIPTION_FILTER_PREFIX = "DatadogSubscriptionFilter";
+
+export enum RuntimeType {
+  NODE,
+  PYTHON,
+  UNSUPPORTED,
+}
+
+export const DefaultDatadogProps = {
+  addLayers: true,
+  enableDatadogTracing: true,
+  injectLogContext: true,
+  enableDatadogLogs: true,
+  architecture: "X86_64",
+  captureLambdaPayload: false,
+};
+
+export enum TagKeys {
+  CDK = "dd_cdk_construct",
+}
+
+export const runtimeLookup: { [key: string]: RuntimeType } = {
+  "nodejs10.x": RuntimeType.NODE,
```

Comment:
this runtime has been deprecated, I think we could get rid of it here

Reply:
removed node10, including changing some unit tests to used node12 instead of 10